src/EditorFeatures/Core.Wpf/Peek/IPeekableItemFactory.cs

```diff
@@ -15,3 +22,11 @@ public interface IPeekableItemFactory
 {
     Task<IEnumerable<IPeekableItem>> GetPeekableItemsAsync(ISymbol symbol, Project project, IPeekResultFactory peekResultFactory, CancellationToken cancellationToken);
 }
+
+[Export(typeof(IPeekableItemFactory)), Shared]
+[method: ImportingConstructor]
+[method: Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
+internal sealed class EditorPeekableItemFactory(
+    IMetadataAsSourceFileService metadataAsSourceFileService,
+    IGlobalOptionService globalOptions,
+    IThreadingContext threadingContext) : PeekableItemFactory(metadataAsSourceFileService, globalOptions, threadingContext);
```

Comment:
core code merged down (as is still exported for *us* to use.  This is just a remnant so xaml can still import this until we move them to use the coe in the ea layer.

Reply:
Can you add a comment to the code here just to make the intent clear? In case something prevents us from doing the follow up and somebody else is scratching their head here.

Reply:
done.